src/BuiltInTools/BrowserRefresh/Urls.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Watch.BrowserRefresh
+{
+    internal static class Urls
```

Comment:
```suggestion
    internal static class HotReloadMiddlewarePaths
```
nit: These are Paths 😄 not Urls

Reply:
Let me do this in a follow up. Trying to avoid re-spinning the build for this.